README.md

```diff
@@ -17,15 +17,9 @@ It is backed by [Swash](https://github.com/dfrg/swash).
 
 ## The Parley text stack
 
-Currently, Parley directly depends on five crates: Fontique, HarfRust, Swash, Skrifa, and Peniko.
+Currently, Parley directly depends on four crates: Fontique, HarfRust, Skrifa, and Swash.
```

Comment:
Maybe we should say "four main crates" or "four key crates" or something.

I don't want to be asserting something which isn't true.

This doesn't block this PR, as e.g. we also depend on hashbrown.